prow/cluster/jobs/istio-releases/daily-release/istio-releases.daily-release.collab-gcp-identity.yaml

```diff
@@ -33,73 +33,49 @@ presubmits:
   istio-releases/daily-release:
 
   - name: daily-unit-tests
-    agent: kubernetes
-    context: prow/daily-unit-tests.sh
```

Comment:
This will change the context name from `prow/daily-unit-tests.sh` to the default of `daily-unit-tests`. You probably don't want to change the context name as that would require a status context migration and changing repo required context settings.
I would leave the `context` field as is for now and just drop the other fields.

Reply:
its automatic we have branch protection set up. but yes I would prefer something like "prow: daily-unit-tests" to be consistent with circle-ci. I'll take your suggestion and do the rename of context in a different PR.

Reply:
Branch protection definitely makes things easier, but you may still need to deprecate any existing statuses for the old context to avoid confusing users. We have this tool for that: https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus